src/app-modules/docker-compose

```diff
@@ -0,0 +1,51 @@
+#!/bin/sh
+
+set -x
+
+docker_compose_cmd=docker-compose
+docker_cmd=docker
+
+app_load() {
+    local url="$1"
+    local input_file="$2"
+
+    $docker_cmd image load < "$input_file"
+}
+
+app_save() {
+    local url="$1"
+    local output_file="$2"
+
+    $docker_cmd image save "$url" -o "$output_file"
+}
+
+app_rollout() {
+    local manifests_dir="$1"
+
+    (cd "$manifests_dir" && $docker_compose_cmd up -d > compose.log 2>&1 3>&1) &
```

Comment:
Can you ELI5 what `3>&1` is doing here?

Reply:
we need to detach the command from the terminal. the `disown` is not supported by `/bin/sh`, `setsid` is not always present (would be another dependency, you know), `nohup` can also be absent. that was the best I could come up with.

Reply:
Yes. And how creating a new file descriptor (`3`) and redirect there stdin (`&1`) detaches the command? (I just don't know)